product_docs/docs/biganimal/release/free_trial/detail/connect_to_a_cluster/connect_using_dbeaver.mdx

```diff
@@ -0,0 +1,42 @@
+---
+title: "Connect to your cluster using DBeaver"
+navTitle: DBeaver
+indexCards: none
+---
+
+!!! Note
+    All of the parameters you need to connect can be found on the portal by selecting the name of your cluster on the [Clusters](https://portal.biganimal.com/clusters) page, then selecting the **Connect** tab.
+
+1. Launch [DBeaver](https://dbeaver.io/).
+2. Select the **New Database Connection** button on the toolbar to open the **Connect to a database** dialog.
+3. Select PostgreSQL and select **Next**.
+
+    You may be prompted to download the PostgreSQL JDBC driver.
+4. On the **Main** tab,
+
+    - Enter your cluster's hostname in the **Host** field.
+
+    - Enter "edb_admin" in the **Database** field.
+    - Enter "edb_admin" in the **Username** field.
+    - Enter your cluster's password in the **Password** field. 
+
+5. On the **SSL** tab,
+
+    - Check the **Use SSL** checkbox.
+
+    - Select "require" for the **SSL mode:** field.
+
+6. On the **PostgreSQL** tab, check the **Show all databases** checkbox.
+
+7. Select the **Test connection ...** button and verify that DBeaver is able to connect to your cluster.
```

Comment:
are steps 7 and 8 on the Postgres tab? If so, they should be in a nested list under step 6 including the Show all databases step

Reply:
Nope, those are global to the dialog.

Reply:
Yes, but documentation does not include them. (Not just at EDB...it's a convention throughout the industry.)